packages/maskbook/src/extension/background-script/EthereumServices/providers/Maskbook.ts

```diff
@@ -51,7 +51,7 @@ export function createWeb3(chainId = currentMaskbookChainIdSettings.value, privK
         const weights = getConstant(CONSTANTS, 'PROVIDER_WEIGHT_LIST', chainId)
         url = urls[weights[SEED]]
     }
-    const provider = createProvider(url)
+    const provider = createProvider(url as string)
```

Comment:
Why this change?

Reply:
My vscode was lintering on this type, and I am fixing it here.